test/integration/configurations/nighthawk_15_listeners_http_origin.yaml

```diff
@@ -536,3 +536,12 @@ static_resources:
             typed_config:
               "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
               dynamic_stats: false
+overload_manager:
+  refresh_interval:
+    seconds: 0
+    nanos: 250000000
+  resource_monitors:
+    - name: "envoy.resource_monitors.global_downstream_max_connections"
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.resource_monitors.downstream_connections.v3.DownstreamConnectionsConfig
+        max_active_downstream_connections: 100000
```

Comment:
How did we choose the value `100000` ? We should add a comment here and in the other files to explain our choice and its significance to the posterity.

Same question about the chosen `refresh_interval`.